.github/workflows/release.yml

```diff
@@ -339,3 +339,68 @@ jobs:
               exit 1
             }
           }
+
+  onedrive:
+    name: OneDrive
+    runs-on: ubuntu-18.04
+    needs: preflight
+
+    steps:
+      - name: Check out Devolutions/actions
+        uses: actions/checkout@v3
+        with:
+          repository: Devolutions/actions
+          ref: master
+          token: ${{ secrets.DEVOLUTIONSBOT_TOKEN }}
+          path: ./.github/workflows
+
+      ## Devolutions Toolbox is required for OneDrive uploading
+
+      - name: Install Devolutions Toolbox
+        uses: ./.github/workflows/toolbox-install
+        with:
+          token: ${{ secrets.DEVOLUTIONSBOT_TOKEN }}
+
+      ## Download back the artifacts produced by the other jobs
+
+      - name: Download artifacts (action)
+        if: needs.preflight.outputs.dl-strategy == 'action'
+        uses: actions/download-artifact@v3
+        with:
+          name: devolutions-gateway
+
+      - name: Download artifacts (cli)
+        if: needs.preflight.outputs.dl-strategy == 'cli'
+        shell: pwsh
+        env:
+          GITHUB_TOKEN: ${{ secrets.DEVOLUTIONSBOT_TOKEN }}
+        run: gh run download ${{ needs.preflight.outputs.run }} -n devolutions-gateway --repo $Env:GITHUB_REPOSITORY
+
+      ## workflow_call: The same artifacts persist across the entire run, so the PowerShell/DevolutionsGateway directory will still exist from the CI workflow
+      - name: Manage artifacts
+        shell: pwsh
+        run: Remove-Item (Join-Path PowerShell DevolutionsGateway) -Recurse -ErrorAction Ignore
```

Comment:
This was from other jobs, I’m not exactly sure if it’s required in this case but just in case

Reply:
It's not needed because in next step you select the files you'll upload, so the presence of the PowerShell/DevolutionsGateway directory doesn't matter. It's harmless to leave it.